sapi/fpm/fpm/fpm_status.c

```diff
@@ -521,7 +592,8 @@ int fpm_status_handle_request(void) /* {{{ */
 					proc.auth_user[0] != '\0' ? proc.auth_user : "-",
 					proc.script_filename[0] != '\0' ? proc.script_filename : "-",
 					proc.request_stage == FPM_REQUEST_ACCEPTING ? cpu : 0.,
-					proc.request_stage == FPM_REQUEST_ACCEPTING ? proc.memory : 0);
+					proc.request_stage == FPM_REQUEST_ACCEPTING ? proc.memory : 0,
+					scoreboard.pool);
```

Comment:
this is specific to prometheus so it shouldn't be probably added for all...

Reply:
Only the current pool will be showed on the status page correct? Because then we can indeed drop it off and it will make things a lot easier

Reply:
I think you will need to use separate spprintf to pass pool multiple times so this will be addressed by that...

Reply:
Ok I'll have a look at that, thanks 👍

Reply:
Sorry commented from the wrong account so needed to remove that account from reviewers.

For reference was saying that it will be fixed by the changes for the spprintf :)